modules/tiles/src/tileset/traversers/tileset-traverser.js

```diff
@@ -252,7 +252,18 @@ export default class TilesetTraverser {
   }
 
   updateTileVisibility(tile, frameState) {
-    tile.updateVisibility(frameState);
+    const viewportIds = [];
+    if (this.options.viewportTraversersMap) {
+      for (const key in this.options.viewportTraversersMap) {
+        const value = this.options.viewportTraversersMap[key];
+        if (value === frameState.viewport.id) {
+          viewportIds.push(key);
```

Comment:
I was kind of wondering if we couldn't just track viewportIds during traversal instead of using the viewportTraverserMap.

Reply:
This is for particular case for showing tiles of viewport A on viewport B.